proto/api/api.proto

```diff
@@ -363,3 +369,20 @@ message GetElectionBucketsRequest{
 message GetElectionBucketsResponse{
   repeated iotextypes.ElectionBucket buckets = 1;
 }
+
+/*
+ * slashing APIs
+ */
+
+message GetKickoutListRequest{
+  uint64 epochNum = 1;
+}
+
+message GetKickoutListResponse{
+  repeated KickoutInfo kickoutList = 1;
+  uint32 intensityRate = 2;
+}
+
+message KickoutInfo{
+  string address = 1;
+}
```

Comment:
@CoderZhi @dustinxie maybe use `ReadState` API to get all data beside basic chain operation data?
otherwise we will out grow number of these getter/setter API not related to chain core logic

Reply:
I think it works out, should I move into ReadState() for this one first?